pkg/image/validation/kubernetes_test.go

```diff
@@ -68,18 +76,27 @@ func TestValidateKubernetes(t *testing.T) {
 						"example.com",
 					},
 				},
-				HelmCharts: []image.HelmChart{
-					{
-						Name:    "",
-						Repo:    "oci://registry-1.docker.io/bitnamicharts/apache",
-						Version: "10.7.0",
+				Helm: image.Helm{
+					Charts: []image.HelmChart{
+						{
+							Name:           "",
+							RepositoryName: "apache-repo",
+							Version:        "10.7.0",
+						},
+					},
+					Repositories: []image.HelmRepository{
+						{
+							Name: "invalid-repo",
```

Comment:
Nit: It's slightly better to use a valid value here e.g. `apache-repo` and another value for `RepositoryName` above e.g. `another-apache-repo`.